### ISSUE TITLE
Any update?

### ISSUE THREAD

**Issue by BartoszPiwek** [darigovresearch/MDN-Web-Docs-Glossary-Flashcards#1]
Can we resrape data from mdn?

**Comment by darigovresearch** [comment]
@BartoszPiwek yeah we currently done this manually, happy to do this again if there has been a lot of updates

**Comment by BartoszPiwek** [comment]
I can help build some kind of scraper for this :)

**Comment by darigovresearch** [comment]
Would suggest parsing a cloned version of the source than scraping the site itself. This could be a small python script that reads the [relevant content](https://github.com/mdn/content/tree/main/files/en-us/glossary), converts the markdown into html and compiles it into a single .csv to do some post-processing ready for conversion into a .apkg file
